controllers/notification/mailgun_notification_delivery_service_test.go

```diff
@@ -186,7 +186,7 @@ func TestMailgunNotificationDeliveryService(t *testing.T) {
 				Template:  "replyto",
 				Context:   notCtx,
 			},
-		})
+		}, "")
```

Comment:
Maybe I'm missing something but I don't see any tests for the stone soup notification environment.

Reply:
nope it is indeed missing, let me add that